Tools/clinic/clinic.py

```diff
@@ -3966,8 +3965,10 @@ def set_template_dict(self, template_dict):
             template_dict['base_type_ptr'] = type_ptr
 
 
-
-def add_c_return_converter(f, name=None):
+def add_c_return_converter(
+        f: type,
+        name: str | None = None
+) -> ConverterType:
```

Comment:
This is incorrect. `ConverterType` is an alias for `Callable[..., CConverterType]`. But the `return_converters` dictionary has subclasses of `CCReturnConverter` as its values, which is a different class.